client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -66,6 +66,11 @@ class EditorMoreOptionsCopyPost extends Component {
 
 	render() {
 		const { translate, siteId } = this.props;
+
+		if ( ! siteId ) {
```

Comment:
Could we be more optimistic here? Seems we could render most all of this component, save for the `<PostSelector />`. Would be nicer to show the label and button, disabled if necessary to avoid having them click only for nothing to happen (or even better, allow the dialog to show empty until the site ID is available, then render `PostSelector`).